### PR TITLE
Add prop className in components

### PR DESCRIPTION
**Pull request by dhruvdutt**
Add `className` prop in "Extending" section of all components

### CODE REVIEW

File: docs/v1.0.0/components/DataController.md [hunk 1]
```diff
@@ -73,13 +73,14 @@ All reactivebase components are `rbc` namespaced.
...
-1. customize the look and feel with `componentStyle`,
+1. customize the look and feel with `className`, `componentStyle`,
 2. update the underlying DB query with `customQuery`,
 3. connect with external interfaces using `beforeValueChange` and `onValueChange`.
 
 ```
 <DataController
   ...
+  className="custom-class"
   componentStyle={{"paddingBottom": "10px"}}
```
Comment (siddharthlatest) [inline docs/v1.0.0/components/DataController.md]: Please rebase it on top of the current master branch. `componentStyle` shouldn't be used.

File: docs/v1.0.0/components/DataController.md [hunk 2]
```diff
@@ -113,6 +114,8 @@ All reactivebase components are `rbc` namespaced.
 />
 ```
 
+- **className** `String` [optional]
```
Comment (siddharthlatest) [inline docs/v1.0.0/components/DataController.md]: No need to mention [optional] as this is in the extending section.

**Comment by dhruvdutt** [comment]
@siddharthlatest Updated.

File: docs/v1.0.0/components/SelectedFilters.md [hunk 3]
```diff
@@ -39,14 +39,17 @@ All reactivebase components are `rbc` namespaced.
...
 <SelectedFilters
+  className="custom-class"
   style={{"paddingBottom": "10px"}}
 />
 ```
 
+- **className** `String`  
+    CSS class to be injected on the component container.
 - **style** `Object`
     CSS styles to be applied to the **TextField** component.
```
[the remark this answers is not in the document]
Comment (dhruvdutt) [reply docs/v1.0.0/components/SelectedFilters.md]: @siddharthlatest Updated.

**Pull request merged**